### PR TITLE
add service files

### LINKED ISSUE #149: Hypernetes install using old Kubernetes rpm
It seems current installation is using k8s 1.2 rpm, which locks to docker 1.10.

This will prevent use from using latest dockerd.

### PR DESCRIPTION
**Pull request by xlgao-zju**
close #149 

Signed-off-by: Xianglin Gao xlgao@zju.edu.cn

### CODE REVIEW

File: hack/hypernetes/hypernetes.sh [hunk 1]
```diff
@@ -96,6 +96,126 @@ region = RegionOne
...
+[Install]
+WantedBy=multi-user.target
+EOF
+
+	cat > /usr/lib/systemd/system/kubelet.service <<EOF
+[Unit]
+Description=Kubernetes Kubelet Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+After=docker.service
+Requires=docker.service
```
Comment (feiskyer) [inline hack/hypernetes/hypernetes.sh]: we don't require docker. those two lines should be removed.

**Review by feiskyer — CHANGES_REQUESTED** [review]

File: hack/hypernetes/hypernetes.sh (hunk 1 above)
Comment (xlgao-zju) [reply hack/hypernetes/hypernetes.sh]: OK. But I will take some time to test this pr because of the poor network environment.

Comment (feiskyer) [reply hack/hypernetes/hypernetes.sh]: @xlgao-zju Yep, network is a big problem. Thanks for the work.

**Comment by xlgao-zju** [comment]
@feiskyer updated. And it seems fine on centos 7.2.

**Review by feiskyer — APPROVED** [review]

**Comment by feiskyer** [comment]
Cool, thanks.

**Pull request merged**